src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectAnalyzersPanel.java

```diff
@@ -0,0 +1,141 @@
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.config.project;
+
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.project.Project;
+import com.intellij.ui.ToolbarDecorator;
+import com.intellij.ui.table.JBTable;
+import com.intellij.util.ui.JBUI;
+import java.awt.Component;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import javax.swing.JPanel;
+import javax.swing.JTable;
+import javax.swing.table.AbstractTableModel;
+import javax.swing.table.DefaultTableCellRenderer;
+import javax.swing.table.TableCellRenderer;
+import org.apache.commons.lang.StringUtils;
+import org.sonarlint.intellij.core.ProjectBindingManager;
+import org.sonarlint.intellij.util.SonarLintUtils;
+import org.sonarsource.sonarlint.core.client.api.connected.LoadedAnalyzer;
+
+public class SonarLintProjectAnalyzersPanel {
+  private static final Logger LOGGER = Logger.getInstance(SonarLintProjectAnalyzersPanel.class);
+  private final Project project;
+  private JBTable analyzerTable;
+  private JPanel panel;
+  private JPanel tablePanel;
+  private Model tableModel;
+
+  public SonarLintProjectAnalyzersPanel(Project project) {
+    this.project = project;
+    reload();
+  }
+
+  public JPanel getPanel() {
+    return panel;
+  }
+
+  public void reload() {
+    ProjectBindingManager bindingManager = SonarLintUtils.get(project, ProjectBindingManager.class);
+    try {
+      Collection<LoadedAnalyzer> loadedAnalyzers = bindingManager.getFacade().getLoadedAnalyzers();
+      tableModel.set(loadedAnalyzers);
+    } catch(Exception e) {
+      LOGGER.error(e);
+      tableModel.set(Collections.emptyList());
+    }
+  }
+
+  private void createUIComponents() {
+    tableModel = new Model();
+    analyzerTable = new JBTable(tableModel);
+    analyzerTable.setShowGrid(false);
+    analyzerTable.setIntercellSpacing(JBUI.emptySize());
+    analyzerTable.getEmptyText().setText("No information");
+    analyzerTable.setDragEnabled(false);
+    analyzerTable.setShowVerticalLines(false);
+    analyzerTable.getTableHeader().setReorderingAllowed(false);
+    analyzerTable.enableInputMethods(false);
+    analyzerTable.setDefaultRenderer(String.class, new NoFocusCellRenderer(new DefaultTableCellRenderer()));
+
+    final ToolbarDecorator decorator = ToolbarDecorator.createDecorator(analyzerTable);
+    tablePanel = decorator.createPanel();
+  }
+
+  private class NoFocusCellRenderer extends DefaultTableCellRenderer {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this a "static" inner class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2694)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This class has 6 parents which is greater than 5 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)